src/interface/menu.rs

```diff
@@ -0,0 +1,67 @@
+use super::{Executable, ship_management};
+
+use crate::store::Session;
+use dialoguer::{theme::ColorfulTheme, Select};
+
+enum MainMenuState {
+    Exit,
+    NewShip,
+    ListShips,
+    DisassembleShip,
+    RenameShip,
+    StartGame,
+}
+
+pub struct MainMenu {
```

Comment:
this file's responsibility is to define the `MainMenu` struct and implementation, so I would rename the file to `main_menu.rs`